fhir-server-test/src/test/resources/test.properties

```diff
@@ -36,7 +36,9 @@ test.kafka.topicName = fhirNotifications
 fhirclient.logging.enabled = false
 
 fhirclient.hostnameVerification.enabled = false
-fhirclient.http.receive.timeout = 60000
+
+# Longer than the default Liberty transaction timeout 120s
+fhirclient.http.receive.timeout = 130000
```

Comment:
i wonder if we should make the same change to the default in fhir-client

Reply:
done:
```
            // Use a default that's longer than the default Liberty transaction timeout
            setHttpTimeout(Integer.parseUnsignedInt(getProperty(PROPNAME_HTTP_TIMEOUT, "130000")));
```